src/components/Book.js

```diff
@@ -1,13 +1,23 @@
+/*eslint-disable */
 import PropTypes from 'prop-types';
 import { useDispatch } from 'react-redux';
-import { removeBook } from '../redux/books/booksSlice';
+import { deleteBook, getBookItems } from '../redux/books/booksSlice';
 
-// genre, title, author,
 function Book({
-  id, title, author, category, percentage, chapter,
+  item_id, title, author, category, percentage, chapter,
 }) {
   const dispatch = useDispatch();
 
+  async function handleRemove(e) {
+    e.preventDefault();
+    try {
+      await dispatch(deleteBook(item_id));
+      await dispatch(getBookItems());
+    } catch (error) {
+      console.log(error)
```

Comment:
- [Optional] Kindly, Do not commit `console.log` to your repo. It’s ugly, it kills performance and it can make confidential data be visible to anyone using the browser tools to look at your website. check the [js best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md) for more info 👍